### ISSUE TITLE
Primary monitor still wrong after install

### ISSUE THREAD

**Issue by rykugur** [zakk4223/hyprXPrimary#4]
Installed plugin via hyprpm:

```
 hyprpm list
 → Repository hyprXPrimary:
  │ Plugin hyprXPrimary
  └─ enabled: true
~                         
```

Relevant config:

```
plugin {
  xwaylandprimary {
    display = DP-2
  }
}
```

And output of `hyprctl monitors`:

```
Monitor HDMI-A-1 (ID 0):
	1920x1080@143.85699 at 3440x0
	description: ASUSTek COMPUTER INC ASUS XG16A 0x000175F5 (HDMI-A-1)
	make: ASUSTek COMPUTER INC
	model: ASUS XG16A
	serial: 0x000175F5
	active workspace: 6 (6)
	special workspace: 0 ()
	reserved: 0 30 0 0
	scale: 1.00
	transform: 1
	focused: yes
	dpmsStatus: 1
	vrr: 0
	activelyTearing: false

Monitor DP-2 (ID 1):
	3440x1440@143.97501 at 0x0
	description: Dell Inc. Dell AW3423DW #G7IYMxgwAA8L (DP-2)
	make: Dell Inc.
	model: Dell AW3423DW
	serial: #G7IYMxgwAA8L
	active workspace: 1 (1)
	special workspace: 0 ()
	reserved: 0 30 0 0
	scale: 1.00
	transform: 0
	focused: no
	dpmsStatus: 1
	vrr: 0
	activelyTearing: false
```

Still having apps launch on my secondary screen (HDMI-A-1) on startup, and many games report incorrect available resoloutions (see dota2 screenshot):

![image](https://github.com/zakk4223/hyprXPrimary/assets/2609898/d222fc15-054c-4a7c-8f4b-ffdb641f7d65)

Did I miss a step?

**Comment by zakk4223** [comment]
Oh, are you running 'hyprpm reload' at startup? Hyprpm has to be told to load the plugins, and the recommended way is to run 'hyprpm reload -n' as an 'exec-once' in your config. I should probably add that to the instructions.

**Comment by rykugur** [comment]
> Oh, are you running 'hyprpm reload' at startup? Hyprpm has to be told to load the plugins, and the recommended way is to run 'hyprpm reload -n' as an 'exec-once' in your config. I should probably add that to the instructions.

Thanks, I was unaware of that. I see now that I indeed missed that on the [docs](https://wiki.hyprland.org/Plugins/Using-Plugins/#installing--using-plugins), my bad!

I added this to my config and confirmed that dota is launching with the correct monitor identified as primary. Now, if I can just figure out why my UI is flickering (not visible on recording software, leads me to believe it's refresh-rate/VRR related), I'm all set.

My startup apps are still starting on my second monitor, but that's incredibly minor and according to a comment of [yours](https://github.com/hyprwm/Hyprland/issues/4144#issuecomment-1858378343) I found, it's due to plugins not loading until after the config is parsed.

I'll close, thanks again!
